test/lint.js

```diff
@@ -56,5 +59,13 @@ if (process.argv[2]) {
 }
 
 if (hasErrors) {
+  console.log("");
+  console.log(`Problems in ${Object.keys(filesWithErrors).length} files:`);
+  for (var file in filesWithErrors) {
```

Comment:
This should probably use `let` instead for consistency sake, and to avoid weird bugs with how `var` behaves.

Reply:
Good point, changed 👍